starwars/src/App.js

```diff
@@ -28,11 +31,39 @@ class App extends Component {
         throw new Error(err);
       });
   };
+//toggleComplete and clear Complete are experimental. Cant get em working yet.
+  toggleComplete = id => {
```

Comment:
ToggleComplete currently is not doing anything. You dont really need a this function since the readme only tells you to render them to the screen.